__tests__/app.test.js

```diff
@@ -100,4 +101,42 @@ describe('GET - /api/reviews', () => {
                 expect(body.msg).toBe("Invalid path");
             })
     });
+});
+
+describe('GET - /api/reviews/:review_id/comments', () => {
+    it('200: GET response with an array of comments for the given review_id', () => {
+        return request(app)
+            .get("/api/reviews/2/comments")
+            .expect(200)
+            .then(({ body }) => {
+                const { comments } = body;
+                expect(comments).toHaveLength(3);
+                comments.forEach((item) => {
+                    expect(item).toMatchObject({
+                        comment_id: expect.any(Number),
+                        body: expect.any(String), votes: expect.any(Number),
+                        author: expect.any(String),
+                        review_id: expect.any(Number),
+                        created_at: expect.any(String)
+                    })
+                })
+                expect(comments).toBeSortedBy("created_at", { descending: true });
+            })
+    });
+    it('400: GET response with error message when ID input is not a number', () => {
+        return request(app)
+            .get("/api/reviews/not-a-number/comments")
+            .expect(400)
+            .then(({ body }) => {
+                expect(body.msg).toBe("Bad request");
+            })
+    });
+    it('400: GET response with error message when passed number doesn`t exist', () => {
+        return request(app)
+            .get("/api/reviews/9999/comments")
+            .expect(400)
+            .then(({ body }) => {
+                expect(body.msg).toBe("Bad request");
+            })
+    });
```

Comment:
Great tests so far Freeman, can you please add a test to check that if you make a request to get the comments of an article that doesn't have any comments, it gives back a 200 and an empty array of comments rather than an error, and get that test passing 🧪